vllm/v1/core/scheduler.py

```diff
@@ -34,7 +34,7 @@ def __init__(
             block_size=self.cache_config.block_size,
             num_gpu_blocks=num_gpu_blocks,
             sliding_window=self.cache_config.sliding_window,
-            enable_caching=True)
+            enable_caching=self.cache_config.enable_prefix_caching)
```

Comment:
@WoosukKwon Should we always turn on prefix caching in V1?

Reply:
We could override the schedule config in v1 for now, but that's a bit hacky, and prefix caching doesn't support VLM in v1 atm

Reply:
As @njhill suggested, I think we can turn it on by default for non-VLMs. (`self.model_config.is_multimoal`)